topics/list_topics.go

```diff
@@ -0,0 +1,35 @@
+package topics
+
+import (
+	"context"
+	"regexp"
+
+	kafkago "github.com/segmentio/kafka-go"
```

Comment:
If we don't rename the package it'll default to `kafka` as the package name which I personally think is fine
```suggestion
	"github.com/segmentio/kafka-go"
```

Reply:
Done